### DISCUSSION TITLE
请问外观接近的字母是否有考虑做区分的计划？

### DISCUSSION THREAD

**Discussion by willre** [atelier-anchor/smiley-sans#146] [General]
我在尝试将smiley-sans  作为一些应用的默认字体，发现 Il1（iL1）、0oO、q9g 这些外观接近的字体并不容易区分，请问是否可以对外观接近的字符加区分标识，或者 是否已经有了作区分的计划了？

**Comment by stone-zeng** [comment]
#12, #28
